src/ui/main.js

```diff
@@ -2,28 +2,35 @@ import { React, useState, useCallback } from "react";
 import { createRoot } from "react-dom/client";
 import Api from "./Api.js";
 import App from "./App.js";
+import Deck from "./Deck.js";
 
 async function main() {
-  const newDeck = await Api.deck();
-  console.log(newDeck);
+  //const newDeck = await Api.deck();
+  //console.log(newDeck);
 
-  const fetchedDeck = await Api.deck("1234");
-  console.log(fetchedDeck);
+  const fetchedDeck = new Deck();
+  await fetchedDeck.getDeck();
```

Comment:
This is a little confusing to me. Seems like we want to do something like this:

const deck = Deck.create(); // here create is a static method on a class that encapsulates the id